pom.xml

```diff
@@ -46,6 +46,7 @@
             <plugin>
                 <groupId>com.powsybl</groupId>
                 <artifactId>powsybl-itools-packager-maven-plugin</artifactId>
+                <version>5.1.1</version>
```

Comment:
I was forced to add powsybl-core version here...

Reply:
Warning below occurs if no version is added. It seems that in plugins does not use the dependencyManagement versions.

> [WARNING] Some problems were encountered while building the effective model for com.powsybl:powsybl-distribution:pom:2023.0.0
[WARNING] 'build.plugins.plugin.version' for com.powsybl:powsybl-itools-packager-maven-plugin is missing. @ line 46, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]